rllib/utils/replay_buffers/utils.py

```diff
@@ -21,6 +21,72 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_batch_indices(batch: SampleBatchType, num_sequences: int) -> np.ndarray:
```

Comment:
can you please write a comprehensive doc string to explain what is going on in this function.